### PR TITLE
feat: world-acl command

### PR DESCRIPTION
**Pull request by marianogoldman**

### CODE REVIEW

**Review by pedrotambo — APPROVED** [review]

File: src/commands/deploy.ts [hunk 1]
```diff
@@ -252,7 +252,8 @@ export async function main(): Promise<void> {
     debug('\n' + error.stack)
     failWithSpinner('Could not upload content', error)
   }
-  return
+
+  process.exit(0)
```
Comment (menduz) [inline src/commands/deploy.ts:256]: 0 means success, we should use the `process.exitCode` by default

```suggestion
  process.exit()
```

Comment (marianogoldman) [reply src/commands/deploy.ts:256]: Removed the 0. Yes, it is a successful exit.

File: src/lib/WorldsContentServer.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    this.options = args
+    this.options.config = this.options.config || getConfig()
+    this.targetContent = args.targetContent
+    if (process.env.DCL_PRIVATE_KEY) {
+      this.createWallet(process.env.DCL_PRIVATE_KEY)
+    }
+  }
+
+  async link(payload: string): Promise<LinkerResponse> {
+    return new Promise<LinkerResponse>(async (resolve, reject) => {
```
Comment (hugoArregui) [inline src/lib/WorldsContentServer.ts:40]: this is a bit confusing to me, do you need a promise, can't you just return `linker.link(this.options.linkerPort!, !!this.options.isHttps)`?

Comment (marianogoldman) [reply src/lib/WorldsContentServer.ts:40]: Well... the resolution of the promise is on a `link:success` event received from the `WorldsContentServerLinkerAPI`. Not sure how to re-write it without a promise...

**Review by hugoArregui — APPROVED** [review]

**Pull request merged**